### ISSUE TITLE
Sentinel 2 Sensor Not Showing in IVVRM

### ISSUE THREAD

**Issue by AlfredLawrence** [EnMAP-Box/enmap-box#1043]
Good morning, 
I apologize if this is not the place to ask this question. I successfully use Enmap Box on a Desktop pc within QGis and now I need to use it on laptop pc as well. However, the Sentinel 2 sensor spectrum in the tool with ProSAIL is not shown and I am getting the following error:
" WARNING    Traceback (most recent call last):
              File "C:\Users\loren\AppData\Roaming\QGIS\QGIS3\profiles\default\python\plugins\enmapboxplugin\enmapbox\apps\lmuvegetationapps\IVVRM\IVVRM_GUI.py", line 656, in 
              lambda: self.select_s2s(sensor_index=self.gui.SType_combobox.currentIndex())) # Sensor Type combobox
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
              File "C:\Users\loren\AppData\Roaming\QGIS\QGIS3\profiles\default\python\plugins\enmapboxplugin\enmapbox\apps\lmuvegetationapps\IVVRM\IVVRM_GUI.py", line 365, in select_s2s
              sensor_init_success = s2s.init_sensor()
              ^^^^^^^^^^^^^^^^^
              File "C:\Users\loren\AppData\Roaming\QGIS\QGIS3\profiles\default\python\plugins\enmapboxplugin\enmapbox\apps\lmuvegetationapps\Resources\Spec2Sensor\Spec2Sensor_core.py", line 80, in init_sensor
              self.fwhm = srf_file['sensor_fwhm'] # deprecated
              ~~~~~~~~^^^^^^^^^^^^^^^
              File "C:\OSGeo4W\apps\Python312\Lib\site-packages\numpy\lib\npyio.py", line 263, in __getitem__
              raise KeyError(f"{key} is not a file in the archive")
             KeyError: 'sensor_fwhm is not a file in the archive'
             "

I have tried several times to reinstall it but it keeps giving me this error. How could I solve it?
I thank you sincerely for your time and apologize for the disturbance
![S2_Spectrum Not Showing Python Error](https://github.com/user-attachments/assets/020be05d-d0ac-400e-a94c-5c0448c4e5b3)
![S2_Spectrum Not Showing](https://github.com/user-attachments/assets/d0b72862-6f9b-4e98-832e-574960194a63)

**Comment by janzandr** [comment]
@steffi1296 can you help?

**Comment by steffi1296** [comment]
Good morning,
thank you for reporting the issue. It has already been fixed and released in version 3.15.2. If you are still using version 3.15.1, please update the EnMAP-Box 3 plugin, and it should resolve the problem
